misc/json-to-record-converter/src/test/resources/json/ChoreoTransPayloads/sample_0_input.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Shall we add a test for JSON field names with special characters and Unicode characters?

Reply:
Yes, will do that